public/app/core/services/panelAttentionSrv.ts

```diff
@@ -0,0 +1,39 @@
+import { PanelAttentionSrv, PanelWithAttention } from '@grafana/runtime';
+import { VizPanel } from '@grafana/scenes';
+
+export class PanelAttentionService implements PanelAttentionSrv<VizPanel> {
+  private panelId: PanelWithAttention = null;
+  private vizPanelWithAttention: VizPanel | null = null;
+
+  getPanelWithAttention(): PanelWithAttention | VizPanel {
+    return this.panelId || this.vizPanelWithAttention;
+  }
+
+  setPanelWithAttention(panelElement: PanelWithAttention | VizPanel): void {
```

Comment:
Consider also using ref.current for the scene use case, to simplify typing in runtime.